### PR TITLE
Update "Tested up to" in Meta

### PR DESCRIPTION
**Pull request by miriamgoldman**
This PR updates the "Tested up to" in the plugin meta to match the most recent version.

### CODE REVIEW

**Comment by jazzsequence** [comment]
Behat tests are failing because the fixture site is using 6.4.1. Should get the test site updated, make sure the Behat tests pass and then apply the change.

**Comment by pwtyler** [comment]
This also needs to be branched to `develop` which in turn is also out of sync with the release. I'll try to do some cleanup today. @jazzsequence you free tomorrow? Maybe we can poke at the docs-only workflow with this project during CMSP work session.

**Comment by jazzsequence** [comment]
Sorry @pwtyler only just seeing this comment 😅. 
I can work on the fixture site today to ensure the version of WP installed is current and then we can merge into `develop`. I can look into the docs-only workflow we have elsewhere.

**Review by jazzsequence — APPROVED** [review]

**Comment by pwtyler** [comment]
OK `develop` has been updated and rebased back on top of `main` which we never did following 1.4.3's release. I think we need to get all of the current CI changes into `main` so that we can then do an asset-only deploy from there to bump tested up to without cutting a release. Clearly we've not smoothed out this process just yet.

**Comment by jazzsequence** [comment]
Well, you'll note that the plugin release actions were never merged to `main` either (probably because we didn't want to cut a release, but maybe that wouldn't have triggered anyway if it was CI-only changes).

File: README.md [hunk 1]
```diff
@@ -2,8 +2,8 @@
 **Contributors:** [getpantheon](https://profiles.wordpress.org/getpantheon), [outlandish josh](https://profiles.wordpress.org/outlandish-josh), [mpvanwinkle77](https://profiles.wordpress.org/mpvanwinkle77), [danielbachhuber](https://profiles.wordpress.org/danielbachhuber), [andrew.taylor](https://profiles.wordpress.org/andrew.taylor), [jazzs3quence](https://profiles.wordpress.org/jazzs3quence), [stovak](https://profiles.wordpress.org/stovak), [jspellman](https://profiles.wordpress.org/jspellman/), [rwagner00](https://profiles.wordpress.org/rwagner00/)  
 **Tags:** comments, sessions  
 **Requires at least:** 4.7  
-**Tested up to:** 6.3  
-**Stable tag:** 1.4.3  
+**Tested up to:** 6.6.2    
+**Stable tag:** 1.4.4-dev
```
Comment (pwtyler) [inline README.md:6]:
```suggestion
**Stable tag:** 1.4.3  
```

File: README.md [hunk 2]
```diff
@@ -98,120 +98,154 @@ Adds a WP-CLI command to add an index to the sessions table if one does not exis
 
 ## Changelog ##
 
+### 1.4.4-dev ###
+
```
Comment (pwtyler) [inline README.md:102]:
```suggestion
```

File: pantheon-sessions.php [hunk 3]
```diff
@@ -1,7 +1,7 @@
 <?php
 /**
  * Plugin Name: Native PHP Sessions for WordPress
- * Version: 1.4.3
+ * Version: 1.4.4-dev
```
Comment (pwtyler) [inline pantheon-sessions.php:4]:
```suggestion
 * Version: 1.4.3
```

File: pantheon-sessions.php [hunk 4]
```diff
@@ -13,7 +13,7 @@
 
 use Pantheon_Sessions\Session;
 
-define( 'PANTHEON_SESSIONS_VERSION', '1.4.3' );
+define( 'PANTHEON_SESSIONS_VERSION', '1.4.4-dev' );
```
Comment (pwtyler) [inline pantheon-sessions.php:16]:
```suggestion
define( 'PANTHEON_SESSIONS_VERSION', '1.4.3' );
```

File: readme.txt [hunk 5]
```diff
@@ -2,8 +2,8 @@
 Contributors: getpantheon, outlandish josh, mpvanwinkle77, danielbachhuber, andrew.taylor, jazzs3quence, stovak, jspellman, rwagner00
 Tags: comments, sessions
 Requires at least: 4.7
-Tested up to: 6.3
-Stable tag: 1.4.3
+Tested up to: 6.6.2
+Stable tag: 1.4.4-dev
```
Comment (pwtyler) [inline readme.txt:6]:
```suggestion
Stable tag: 1.4.3
```

File: readme.txt [hunk 6]
```diff
@@ -96,6 +96,8 @@ Adds a WP-CLI command to add an index to the sessions table if one does not exis
 
 == Changelog ==
 
+= 1.4.4-dev =
+
```
Comment (pwtyler) [inline readme.txt:100]:
```suggestion
```

**Comment by pwtyler** [comment]
Final product of this PR to be merged to 'release':
- Bump "tested up to"
- Additional checks for asset-only release

**Pull request merged**